lisa/schema.py

```diff
@@ -1155,6 +1155,8 @@ def reload_requirements(self) -> None:
 
         if self.nodes_raw:
             for node_raw in self.nodes_raw:
+                if not isinstance(node_raw, dict):
```

Comment:
to_dict is handled by load_by_type already.

Reply:
I saw it threw exception here `node_raw[constants.TYPE]` when pass in as list.